shell/src/main/java/tachyon/shell/command/RmCommand.java

```diff
@@ -44,24 +42,17 @@ public String getCommandName() {
 
   @Override
   void runCommand(TachyonURI path) throws IOException {
-    TachyonFile fd;
-    FileInfo fInfo;
     try {
-      fd = mTfs.open(path);
-      fInfo = mTfs.getInfo(fd);
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-
-    if (fInfo.isFolder) {
-      throw new IOException("rm: cannot remove a directory, please try rmr <path>");
-    }
-
-    try {
-      mTfs.delete(fd);
+      if (!mTfs.exists(path)) {
```

Comment:
Good idea, I left a TODO here because the exception messages don't make too much sense yet, since we haven't moved completely to communicating with TachyonURIs with the master.